WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -428,21 +430,26 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String permissi
             grantResults) {
         switch (requestCode) {
             case CAMERA_AND_MEDIA_PERMISSION_REQUEST_CODE:
-                for (int grantResult : grantResults) {
-                    if (grantResult == PackageManager.PERMISSION_DENIED) {
-                        ToastUtils.showToast(this.getActivity(), getString(R.string
-                                .gravatar_camera_and_media_permission_required), ToastUtils.Duration.LONG);
-
-                        AnalyticsTracker.track(AnalyticsTracker.Stat
-                                .ME_GRAVATAR_PERMISSIONS_DENIED);
-
-                        return;
+                if (permissions == null || permissions.length == 0) {
+                    AnalyticsTracker.track(AnalyticsTracker.Stat.ME_GRAVATAR_PERMISSIONS_INTERRUPTED);
+                }  else {
+                    for (int i = 0; i < grantResults.length; i++) {
+                        if (grantResults[i] == PackageManager.PERMISSION_DENIED) {
+                            ToastUtils.showToast(this.getActivity(), getString(R.string
+                                    .gravatar_camera_and_media_permission_required), ToastUtils.Duration.LONG);
+
+                            Map<String, Object> properties = new HashMap<>();
+                            properties.put("permission denied", permissions[i]);
+                            AnalyticsTracker.track(AnalyticsTracker.Stat.ME_GRAVATAR_PERMISSIONS_DENIED, properties);
+
+                            return;
```

Comment:
Since this returns here, only the first denied permission is tracked. Do we want to track when both permissions (`WRITE_EXTERNAL_STORAGE` and `CAMERA`) are denied?